src/sentry/static/sentry/app/components/forms/textareaField.tsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import InputField from 'app/components/forms/inputField';
 
-export default class TextareaField extends InputField {
+type State = InputField['state'] & {
+  value: string | undefined;
```

Comment:
Isn't this the same as `value?: string`?

Reply:
@billyvg I think `value?` means the key might not exist in state. `value: string | undefined;` means the the key `value` exists but the value might be `undefined`.